examples/mugen/data/mugen_datamodules.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Callable, NamedTuple, Optional
+
+import pytorch_lightning as pl
+import torch
+import torch.distributed as dist
+import torch.utils.data as data
+
+from .mugen_dataset import MUGENDataset, MUGENDatasetArgs
+
+
+class MUGENDataModuleArgs(NamedTuple):
```

Comment:
why make this class, how about just using these args in `MUGENDataModule` itself? [FLAVA](https://github.com/facebookresearch/multimodal/blob/a1c136853ebee87bb148e2442991f5430cb40fa2/examples/flava/data/datamodules.py#L60) does something similar